cloud: update Request Units per Second metric for Premium (#22878)
pingcap/docs: tidb-cloud/premium/built-in-monitoring-premium.md

| Metric name  | Labels | Description                                   |
| :------------| :------| :-------------------------------------------- |

> | Request Units per Second | Total RU/s, AVG RU/s | The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. `Total RU/s` shows real-time RU consumption per second. `AVG RU/s` shows the average RU per second over the selected time range, giving you a clearer view of resource consumption. In addition to queries that you run, request units can be consumed by background activities, so when the QPS is 0, the request units per second might not be zero. |

Author: ```suggestion
| Request Units per Second | Total RU per second, AVG RU/s | The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. `Total RU per second` shows real-time RU consumption per second. `AVG RU/s` shows the average RU per second over the selected time range, giving you a clearer view of resource consumption. In addition to queries that you run, request units can be consumed by background activities, so when the QPS is 0, the request units per second might not be zero. |
```

> | Request Units per Second | Total RU per second, AVG RU/s | The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. `Total RU per second` shows real-time RU consumption per second. `AVG RU/s` shows the average RU/s over the selected time range, giving you a clearer view of overall resource consumption. In addition to queries that you run, request units can be consumed by background activities, so when the QPS is 0, the request units per second might not be zero. |

Author: ```suggestion
| Request Units per Second | Total RU per second, AVG RU per second | The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. `Total RU per second` shows real-time RU consumption per second. `AVG RU per second` shows the average RU per second over the selected time range, giving you a clearer view of resource consumption. In addition to queries that you run, request units can be consumed by background activities, so when the QPS is 0, the request units per second might not be zero. |
```

> | Request Units per Second | Total RU per second, AVG RU/s | The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. `Total RU per second` shows real-time RU consumption per second. `AVG RU/s` shows the average RU per second over the selected time range, giving you a clearer view of resource consumption. In addition to queries that you run, request units can be consumed by background activities, so when the QPS is 0, the request units per second might not be zero. |

Reviewer: ```suggestion
| Request Units per Second | Total RU per second, AVG RU/s | The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. `Total RU per second` displays the real-time RU consumption per second. `AVG RU/s` displays the average RU consumption per second over the selected time range, helping you better understand resource consumption. In addition to queries that you run, background activities can also consume RUs. Therefore, when the QPS is 0, the RU consumption per second might still be greater than 0. |
```

> | Request Units per Second | Total RU per second, AVG RU per second | The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. `Total RU per second` shows real-time RU consumption per second. `AVG RU per second` shows the average RU per second over the selected time range, giving you a clearer view of resource consumption. In addition to queries that you run, request units can be consumed by background activities, so when the QPS is 0, the request units per second might not be zero. |

Author: ```suggestion
| Request Units per Second | Total RU/s, AVG RU/s | The Request Unit (RU) is a unit of measurement used to track the resource consumption of a query or transaction. `Total RU/s` shows real-time RU consumption per second. `AVG RU/s` shows the average RU per second over the selected time range, giving you a clearer view of resource consumption. In addition to queries that you run, request units can be consumed by background activities, so when the QPS is 0, the request units per second might not be zero. |
```
